fatiando/mesher/mesh.py

```diff
@@ -388,6 +388,117 @@ def copy(self):
         return cp.deepcopy(self)
 
 
+class PointMesh(object):
+    """
+    A generic mesh of 3D point sources (spheres of unit volume).
+
+    Use this as a 1D list of :class:`~fatiando.mesher.Sphere`.
+
+    Grid points are ordered like a C matrix, first each row in a column, then
+    change columns. In this case, the x direction (North-South) are the rows
+    and y (East-West) are the columns.
+
+    Parameters:
+
+    * x, y, z : 1d-arrays
+        The x, y, and z coordinates of each point in the mesh
+        (remember, z is positive downward).
```

Comment:
Please include a note reminding that x points North and y points East.

Reply:
Included